.github/workflows/run-tests.yml

```diff
@@ -18,15 +19,34 @@ jobs:
           java-version: 11
           distribution: temurin
           cache: maven
+      - name: Check formatting in core
+        run: mvn speedy-spotless:check --batch-mode --no-transfer-progress -q -f ./dhis-2/pom.xml
+      - name: Check formatting in web
+        run: mvn speedy-spotless:check --batch-mode --no-transfer-progress -q -f ./dhis-2/dhis-web/pom.xml
+      - name: Check maven dependencies
+        run: mvn dependency:analyze --threads 2C --batch-mode -no-transfer-progress --update-snapshots -f ./dhis-2/pom.xml
```

Comment:
I remember you saying dhis-web/pom.xml is a separate root pom. So currently we would not analyze its dependencies, right? If so I think we should we also run the dependency:analyze on it.

Reply:
We should, but to do so we need to fix them before.
I am not including it as part of this PR